neofs-testlib/neofs_testlib/env/env.py

```diff
@@ -431,40 +463,49 @@ def start(self):
             config_path=self.network_config,
             custom=Path(network_config_template).is_file(),
             morph_endpoint=self.rpc_address,
-            alphabet_wallets_path=self.alphabet_wallet.path,
+            alphabet_wallets_path=self.neofs_env.alphabet_wallets_dir,
             default_password=self.neofs_env.default_password,
         )
-        logger.info("Generating alphabet wallets")
-        self.neofs_env.generate_wallet(WalletType.ALPHABET, self.alphabet_wallet, network_config=self.network_config)
-        logger.info(f"Generating IR config at: {self.ir_node_config_path}")
 
+    @allure.step("Start Inner Ring node")
+    def start(self, wait_until_ready=True):
+        logger.info(f"Generating IR config at: {self.ir_node_config_path}")
         ir_config_template = "ir.yaml"
 
+        pub_keys_of_existing_ir_nodes = [
+            wallet_utils.get_last_public_key_from_wallet(ir_node.alphabet_wallet.path, ir_node.alphabet_wallet.password)
+            for ir_node in self.neofs_env.inner_ring_nodes
+        ]
+
+        seed_node_addresses_of_existing_ir_nodes = [ir_node.p2p_address for ir_node in self.neofs_env.inner_ring_nodes]
+
         NeoFSEnv.generate_config_file(
             config_template=ir_config_template,
             config_path=self.ir_node_config_path,
             custom=Path(ir_config_template).is_file(),
             wallet=self.alphabet_wallet,
-            public_key=wallet_utils.get_last_public_key_from_wallet(
-                self.alphabet_wallet.path, self.alphabet_wallet.password
-            ),
+            public_keys=pub_keys_of_existing_ir_nodes,
             ir_storage_path=self.ir_storage_path,
-            seed_nodes_address=self.seed_nodes_address,
+            seed_nodes_addresses=seed_node_addresses_of_existing_ir_nodes,
             rpc_address=self.rpc_address,
             p2p_address=self.p2p_address,
             grpc_address=self.grpc_address,
             ir_state_file=self.ir_state_file,
+            peers_min_number=int(
+                len(self.neofs_env.inner_ring_nodes) - (len(self.neofs_env.inner_ring_nodes) - 1) / 3 - 1
+            ),
+            set_roles_in_genesis=str(False if len(self.neofs_env.inner_ring_nodes) == 1 else True).lower(),
```

Comment:
I'd expect it to be used for all networks, but maybe this condition would be useful to at least test things in absence of the option.

Reply:
So, leave it as is?